src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+~NuGet.Protocol.Core.Types.SourceRepository.GetServiceIndexV3FromCache(out NuGet.Protocol.ServiceIndexResourceV3 result) -> bool
```

Comment:
This API doesn't really seem ideal.
It's referring to a specific resource while SourceRepository tends to be a more generic resource.

Reply:
Maybe GetResourceFromCacheInstead? and allow any resource type that has been downloaded to be retrieved?